src/core/messenger.ts

```diff
@@ -500,24 +491,12 @@ export const unregister: UnregisterListener = (type, callback, options) => {
  * @param persistentListeners The persistent listener registration functions
  */
 export const init = (
-	listeners: Array<
-		(
-			register: RegisterListener,
-			errorHandler: MessengerErrorHandler,
-		) => void
-	>,
-	persistentListeners: Array<
-		(
-			register: RegisterPersistentListener,
-			errorHandler: MessengerErrorHandler,
-		) => void
-	>,
-	errorHandler: MessengerErrorHandler,
+	listeners: Array<(register: RegisterListener) => void>,
+	persistentListeners: Array<(register: RegisterPersistentListener) => void>,
 ): void => {
-	reportError = errorHandler;
-	listeners.forEach((moduleInit) => moduleInit(register, errorHandler));
+	listeners.forEach((moduleInit) => moduleInit(register));
 	persistentListeners.forEach((moduleInit) =>
-		moduleInit(registerPersistentListener, errorHandler),
```

Comment:
None of the listeners were actually using the `errorHandler` aka `reportError` so this has been removed